proxy/http/Http1ClientTransaction.cc

```diff
@@ -67,6 +67,7 @@ Http1ClientTransaction::transaction_done()
   // If the parent session is not in the closed state, the destroy will not occur.
   if (parent) {
     parent->destroy();
+    parent = NULL;
```

Comment:
Yes, destroy doesn't necessarily destroy.  But from transaction_done(), the client transaction is done with the parent session.  So even if the parent is still around, the client transaction should no longer be manipulating it.

The change looks good to me.